src/main/java/hudson/plugins/analysis/core/BuildHistory.java

```diff
@@ -47,7 +53,7 @@
      *            reference builds or not
      * @since 1.66
```

Comment:
Yes. it should be changed. Fixing (not only here, in all places where the javadoc annotation already existed and require update).